steps/recursive-exception-reports.sh

```diff
@@ -17,28 +17,22 @@ else
 fi
 
 echo -n "[*] Populating staging table and exporting CSV... "
-#psql -h "$DATA_STORE_HOST" -p "$DATA_STORE_PORT" -d "$DATA_STORE_NAME" -U "$DATA_STORE_USER" -w -v FROM_DATE="'${FROM_DATE}'" -v JURISDICTION="'${JURISDICTION}'" -f scripts/migrate-staging.sql data_store_snapshot 2>&1 > /dev/null
 
 if [ $OPERATION$DBTYPE = "exportrecursiveexceptionrealtime" ]; then
     echo "EXPORTING Recursive DOCUMENT IDs : Exporting Document Ids from Temp DB $DATA_STORE_HOST  $DATA_STORE_PORT $DATA_STORE_NAME $DATA_STORE_USER"
     export PGPASSWORD="$DATA_STORE_PASS"
```

Comment:
May be -W option is better without even variable $DATA_STORE_PASS ??

Reply:
I removed it because otherwise you are prompted to enter the password for every invocation of `psql`